fluentlenium-core/src/main/java/org/fluentlenium/configuration/DefaultWebDriverFactories.java

```diff
@@ -0,0 +1,37 @@
+package org.fluentlenium.configuration;
+
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class DefaultWebDriverFactories {
+    public static class FirefoxWebDriverFactory extends ReflectiveWebDriverFactory {
+        public FirefoxWebDriverFactory() {
+            super("firefox", "org.openqa.selenium.firefox.FirefoxDriver");
```

Comment:
why we are adding drivers by reflection instead of creating new objects here of particular webdriver?